# HG changeset patch
# Parent ebfc652c7c002909bf4a544b9e8cdb1deed148fb
Let the timer code restart the discovery on a write error

This avoid stupid infinite loops on ENETUNREACH for example.



diff --git a/lifx/broadcast.c b/lifx/broadcast.c
--- a/lifx/broadcast.c
+++ b/lifx/broadcast.c
@@ -240,7 +240,6 @@
 error_reset:
     lgtd_lifx_broadcast_close();
     lgtd_lifx_broadcast_setup();
-    lgtd_lifx_broadcast_discovery();
 }
 
 void
